content/authors/hactar/_index.html

```diff
@@ -0,0 +1,4 @@
+---
+title: Patrick W
```

Comment:
I think his full name is Patrick W. Best https://www.linkedin.com/in/patrick-w-best-92938b4/

Consider using that for consistency.

Reply:
That is another Patrick. But let's ask him directly @hactar are you OK if we add your full name or do you prefer appearing as Patrick W?

Reply:
Hehe, thats not me. I'm Patrick Wolowicz, you're welcome to use my full name if you would like ☺️

Reply:
Haha another Patric W at another company called subzero.